### PR TITLE
doc/ko: Add missing options in man pages

### PR DESCRIPTION
**Pull request by jeonghwanin**
The missing options -Z *SIZE*, \--size-filter=*SIZE*,
-L LOCATION' '--loc-filter=LOCATION' and '--with-syms=DIR'
are translated and added to man pages in Korean.

Signed-off-by: Jeonghwan In <jhpc0128@naver.com>

### CODE REVIEW

**Comment by honggyukim** [comment]
Thanks for the PR, but please rewrite the commit message so that anyone can understand what this patch is about only with the commit message without looking into the changes in diff.

**Comment by MichelleJin12** [comment]
@jeonghwanin 
How about this?
```
doc/ko:Update uftrace-graph.md in Korean

The missing options '-L LOCATION', '--loc-filter=LOCATION' and
'--with-syms=DIR' are translated and added to uftrace-graph.md in Korean.

Signed-off-by: Jeonghwan In <jhpc0128@naver.com>
```

You can rewrite the commit message by
```sh
git commit --amend
# rewrite commit message, save and exit the text editor
git push origin doc --force
```
and then rewrite the PR body, too.

**Comment by jeonghwanin** [comment]
Thank you so much for your guidance! I really appreciate your help in advising me to edit the commit message.

File: doc/ko/uftrace-graph.md [hunk 1]
```diff
@@ -91,6 +95,8 @@ GRAPH 옵션
 :   타입(TYPE)으로 일치하는 패턴을 보여준다. 가능한 형태는 `regex`와 `glob`이다.
     기본 설정은 `regex`이다.
 
+\--with-syms=*DIR*
+:   DIR 디렉토리의 .sym 파일에서 심볼(symbol) 데이터를 읽는다. 이는 심볼(symbol) 데이터가 제거된 바이너리 파일을 다루는데 유용하다. 바이너리 파일 이름은 저장할 때와 사용할 때 동일해야 한다.
```
Comment (namhyung) [inline doc/ko/uftrace-graph.md:99]: Please break the long line (around 80 characters).

Also please check the other man pages for the same option.  I'd like to have them with the same translation.

Comment (jeonghwanin) [reply doc/ko/uftrace-graph.md:99]: Ok! I'll take it.

**Comment by honggyukim** [comment]
I have one more change request.
```diff
-doc/ko:Update uftrace-graph.md in Korean
+doc/ko: Add missing options in uftrace-graph.md
```

**Comment by jeonghwanin** [comment]
I add common same option in other man pages in korean.
I change commit message to '[doc/ko:Add missing options in man pages in Korean]'.
thank you.

**Comment by jeonghwanin** [comment]
I add common same option in other man pages in korean.
I change commit message to '[doc/ko:Add missing options in man pages in Korean]'.
thank you.

**Comment by honggyukim** [comment]
We and other projects mostly have a single whitespace after the `:` so please change the title as follows.
```diff
-doc/ko:Add missing options in man pages in Korean
+ doc/ko: Add missing options in uftrace-graph.md
```
And “man page” doesn’t provide useful information because we already have “doc/ko” part.

In addition, “in Korean” part is not needed because we have “doc/ko” part again.

**Review by honggyukim — APPROVED** [review]
LGTM. Thanks for finding all the missing updates in Korean man pages.

**Pull request merged**